src/main/java/wooteco/subway/service/LineService.java

```diff
@@ -0,0 +1,63 @@
+package wooteco.subway.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+
+import wooteco.subway.dao.LineDao;
+import wooteco.subway.domain.Line;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+
+@Service
+public class LineService {
+    
+    private final LineDao lineDao;
+    
+    public LineService(LineDao lineDao){
+        this.lineDao = lineDao;
+    }
+    public LineResponse save(LineRequest lineRequest) {
```

Comment:
```suggestion
    public LineService(LineDao lineDao) {
        this.lineDao = lineDao;
    }

    public LineResponse save(LineRequest lineRequest) {
```

전체적으로 Reformat Code 돌려 주세요.

Reply:
항상 파일에서 단축키를 사용해서 정렬하였는데 패키지 전체에 대해 Reformat Code도 Intellij에서 지원하는군요..! 다음부터 PR전 꼭 수행하겠습니다!